### DISCUSSION TITLE
网页显示不正常，应该是浏览器问题，但不知道怎么解决

### DISCUSSION THREAD

**Discussion by Tonmyn** [bs-community/blessing-skin-server#382] [Q&A] [answered]
根据教程搭建的皮肤站，有时会显示不正常

后面我发现，当出现这个问题时，登陆其他的比较知名的皮肤站也会出现这个问题，Edge ，chrome 浏览器 都试过会这样，显示正常时所有浏览器也会显示正常。

下图为显示不正常时的截图：
--------------------------------------------------------------------------------
![微信图片_20220316173127](https://user-images.githubusercontent.com/18096197/158560563-31f5d7a3-5d15-40be-a7c1-a1897400197f.png)
-------------------------------------------------------------------------------
如果是样式失效，但链接里面并没有包含“index.php”，所以不知道该如何是好

**Comment by Big-Cake-jpg** [comment, marked as answer]
jsDelivr 的锅，建议更换 DNS 或者直接用代理
